specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -499,15 +594,680 @@
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type":"object",
+      "x-ms-discriminator-value": "Volume",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "CubicMeter",
+            "CubicCentimeter",
+            "CubicMillimeter",
+            "Hectoliter",
+            "Decaliter",
+            "Liter",
+            "Centiliter",
+            "Milliliter",
+            "CubicYard",
+            "CubicInch",
+            "CubicFoot",
+            "CubicMile",
+            "FluidOunce",
+            "Teaspoon",
+            "Tablespoon",
+            "Pint",
+            "Quart",
+            "Cup",
+            "Gill",
+            "Pinch",
+            "FluidDram",
+            "Barrel",
+            "Minim",
+            "Cord",
+            "Peck",
+            "Bushel",
+            "Hogshead",
+            "Unspecified"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true
+          },
+          "description": "The Volume Unit of measurement"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "SpeedResolution":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type":"object",
+      "x-ms-discriminator-value": "Speed",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "MetersPerSecond",
+            "KilometersPerHour",
+            "KilometersPerMinute",
+            "KilometersPerSecond",
+            "MilesPerHour",
+            "Knot",
+            "FootPerSecond",
+            "FootPerMinute",
+            "YardsPerMinute",
+            "YardsPerSecond",
+            "MetersPerMillisecond",
+            "CentimetersPerMillisecond",
+            "KilometersPerMillisecond",
+            "Unspecified"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "AreaResolution":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type":"object",
+      "x-ms-discriminator-value": "Area",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "SquareKilometer",
+            "SquareHectometer",
+            "SquareDecameter",
+            "SquareDecimeter",
+            "SquareMeter",
+            "SquareCentimeter",
+            "SquareMillimeter",
+            "SquareInch",
+            "SquareFoot",
+            "SquareMile",
+            "SquareYard",
+            "Acre",
+            "Unspecified"
```

Comment:
Curious: what are our pans to cover units of measure from other cultures? Off the top of my head, Japan at least has some units that are unique and still in use (like the unit of measurement used for floor space).

Reply:
From the API point of view the design is extensible. But, yes, for each language/culture that we support right now, we should add the local measurements to the respective Text recognizer and/or the ML recognizer.